src/js/edu-benefits/1995/config/form.js

```diff
@@ -108,18 +108,17 @@ const formConfig = {
       title: 'Benefit Selection',
       pages: {
         benefitSelection: {
-          title: 'Benefit selection',
+          title: 'Education benefit',
           path: 'benefits-eligibility/benefit-selection',
```

Comment:
Missed this before. We could change the url here, since this doesn't make that much sense now. Maybe just `benefit` or `education-benefit`?